### PR TITLE
eslint configure

### PR DESCRIPTION
**Pull request by AlfredWei0420**

### CODE REVIEW

**Review by beltekylevi — CHANGES_REQUESTED** [review]
It should use the Google's JavaScript style guide! (ESLint can set that up for you, if you select it -- might not work in Git Bash on Windows so try to use the default command line)

https://google.github.io/styleguide/jsguide.html

Also create a task called lint to run the linter. (Basically if I type `npm run lint` in the terminal it should run the linter)

**Review by AlfredWei0420 — COMMENTED** [review]
made some corrections

File: index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'use strict';
```
Comment (beltekylevi) [inline index.js]: This file is unnecessary, please remove

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "epam-jsa-black-pearl",
+  "version": "1.0.0",
+  "description": "Black Pearl workspace -- Chris Alex Alfred",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/.bin/eslint index.js"
```
Comment (beltekylevi) [inline package.json]: You can just use `eslint` here, node will take care of the rest.
It should also check all the javascript files not only the `index.js`

**Review by beltekylevi — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (AlfredWei0420) [reply index.js]: removed

File: package.json (hunk 2 above)
Comment (AlfredWei0420) [reply package.json]: remove uncessary code

**Review by AlfredWei0420 — COMMENTED** [review]
plz review

File: package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "epam-jsa-black-pearl",
+  "version": "1.0.0",
+  "description": "Black Pearl workspace -- Chris Alex Alfred",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/.bin/eslint"
```
Comment (beltekylevi) [inline package.json]: Here you can simply use `"eslint"`

**Review by beltekylevi — CHANGES_REQUESTED** [review]
Did you try it out?

File: package.json (hunk 3 above)
Comment (AlfredWei0420) [reply package.json]: done, it works

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "google"
```
Comment (beltekylevi) [inline .eslintrc.js]: 2 spaces for indentation. (this is also a js file)

**Review by beltekylevi — CHANGES_REQUESTED** [review]
If i execute `npm run lint` it shows me this, it does not run eslint but gives me an instruction how to run it

```bash
epam-jsa-black-pearl (alfredwei) $ npm run lint

> epam-jsa-black-pearl@1.0.0 lint C:\Users\beltekylevi\Documents\workspace\epam-jsa\epam-jsa-black-pearl
> eslint

eslint [options] file.js [file.js] [dir]

Basic configuration:
  -c, --config path::String    Use configuration from this file or shareable config
  --no-eslintrc                Disable use of configuration from .eslintrc
  --env [String]               Specify environments
  --ext [String]               Specify JavaScript file extensions - default: .js
  --global [String]            Define global variables
  --parser String              Specify the parser to be used
  --parser-options Object      Specify parser options

Caching:
  --cache                      Only check changed files - default: false
  --cache-file path::String    Path to the cache file. Deprecated: use --cache-location - default: .eslintcache
  --cache-location path::String  Path to the cache file or directory

Specifying rules and plugins:
  --rulesdir [path::String]    Use additional rules from this directory
  --plugin [String]            Specify plugins
  --rule Object                Specify rules

Ignoring files:
  --ignore-path path::String   Specify path of ignore file
  --no-ignore                  Disable use of ignore files and patterns
  --ignore-pattern [String]    Pattern of files to ignore (in addition to those in .eslintignore)

Using stdin:
  --stdin                      Lint code provided on <STDIN> - default: false
  --stdin-filename String      Specify filename to process STDIN as

Handling warnings:
  --quiet                      Report errors only - default: false
  --max-warnings Int           Number of warnings to trigger nonzero exit code - default: -1

Output:
  -o, --output-file path::String  Specify file to write report to
  -f, --format String          Use a specific output format - default: stylish
  --color, --no-color          Force enabling/disabling of color

Miscellaneous:
  --init                       Run config initialization wizard - default: false
  --fix                        Automatically fix problems
  --debug                      Output debugging information
  -h, --help                   Show help
  -v, --version                Output the version number
  --no-inline-config           Prevent comments from changing config or rules
  --print-config path::String  Print the configuration for the given file
```

**Review by beltekylevi — APPROVED** [review]

**Pull request merged**